packages/integrations/node/README.md

```diff
@@ -119,6 +119,25 @@ app.use(ssrHandler);
 app.listen({ port: 8080 });
 ```
 
+Additionally, you can also pass in an object to be accessed with `Astro.locals` or in Astro middleware:
+
+```js
+import express from 'express';
+import { handler as ssrHandler } from './dist/server/entry.mjs';
+
+const app = express();
+app.use(express.static('dist/client/'))
+app.use((req, res, next) => {
+  const locals = {
+    foo: 'bar'
```

Comment:
Would love if this didn't use `foo: bar` but instead something real?  (In docs, we've been using `title: "New title"` to demonstrate updating a property.)

But, non-blocking!